### PR TITLE
Add nidigital clock generator and frequency counter system tests

### PR DESCRIPTION
**Pull request by ni-jfitzger**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

- [x] ~~I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~~

- [x] I've added tests applicable for this pull request

### What does this Pull Request accomplish?

* Adds nidigital system test `test_clock_generator_abort`
* Adds nidigital system test `test_clock_generator_generate_clock`
* Adds nidigital system test `test_frequency_counter_measure_frequency`

### What testing has been done?

Ran the new system tests on a system with nidigital 19.0.1 installed.

### CODE REVIEW

File: src/nidigital/system_tests/test_system_nidigital.py [hunk 1]
```diff
@@ -558,6 +558,27 @@ def test_ppmu_source(multi_instrument_session):
...
+
+def test_clock_generator_generate_clock(multi_instrument_session):
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+    multi_instrument_session.pins['site0/PinA', 'site1/PinC'].clock_generator_generate_clock(
+        1e6,
+        True)
+
+
+def test_frequency_counter_measure_frequency(multi_instrument_session):
+    measurement_time = datetime.timedelta(milliseconds=5)
```
Comment (sbethur) [inline src/nidigital/system_tests/test_system_nidigital.py:574]: You can get rid of this variable

File: src/nidigital/system_tests/test_system_nidigital.py [hunk 2]
```diff
@@ -558,6 +558,27 @@ def test_ppmu_source(multi_instrument_session):
...
 
+def test_clock_generator_abort(multi_instrument_session):
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+    multi_instrument_session.pins['site0/PinA', 'site1/PinC'].clock_generator_abort()
+
+
+def test_clock_generator_generate_clock(multi_instrument_session):
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+    multi_instrument_session.pins['site0/PinA', 'site1/PinC'].clock_generator_generate_clock(
+        1e6,
```
Comment (sbethur) [inline src/nidigital/system_tests/test_system_nidigital.py:569]: Use a float here

Comment (ni-jfitzger) [reply src/nidigital/system_tests/test_system_nidigital.py:569]: 1e6 is a float.
Run `type(1e6)` and you'll see.

File: src/nidigital/system_tests/test_system_nidigital.py (hunk 1 above)
Comment (ni-jfitzger) [reply src/nidigital/system_tests/test_system_nidigital.py]: I know.
It was just there to keep the code shorter.

**Pull request merged**